app/models/appointment.rb

```diff
@@ -37,14 +37,8 @@ class Appointment < ApplicationRecord
 
   def self.needing_reminder
     pending
-      .includes(:activities)
-      .where(start_at: Time.zone.now..24.hours.from_now)
-      .where.not(
-        email: '',
-        activities: {
-          type: 'ReminderActivity', created_at: 12.hours.ago..12.hours.from_now
-        }
-      )
+      .where(start_at: Time.zone.now..(Time.zone.now + 48.hours))
```

Comment:
Yeah, tbh my first draft just had it all in a single method with local vars of `now`, `two_days_ago` and `two_days_from_now`.

Rubocop didn't like it but I thought it was decent.

Then after breaking it up into two methods I didn't like assigning the local vars that only got called the one time so went back to what we have now.